src/services/storage-service.js

```diff
@@ -298,7 +299,8 @@ export default class StorageService {
 
   async _uploadFiles(files, parentPath) {
     const cdnPathByOriginalName = files.reduce((map, file) => {
-      map[file.originalname] = componseUniqueFileName(file.originalname, parentPath);
+      const uniqueFileName = createUniqueStorageFileName(file.originalname);
+      map[file.originalname] = parentPath ? urlUtils.concatParts(parentPath, uniqueFileName) : uniqueFileName;
```

Comment:
You could remove the ternary operator here, `urlUtils.concatParts` is smart enough to take care of empty parts.

Reply:
Indeed, good point, changed!